ProjectFiles/Assets/Scripts/Network/Events/ServerEliminatePlayersEvent.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.Collections.Generic;
+using Unity.Networking.Transport;
+
+namespace Network.Events
+{
+    public class ServerEliminatePlayersEvent : Event
+    {
+        public byte RoundNumber { get; private set; }
+        public List<byte> Players { get; private set; }
+
+        public ServerEliminatePlayersEvent()
+        {
+            ID = EventType.ServerEliminatePlayersEvent;
+            Length = 1;
+        }
+
+        public ServerEliminatePlayersEvent(byte roundNumber, List<byte> players)
+        {
+            RoundNumber = roundNumber;
+            Players = players;
+            Length = sizeof(byte) * (Players.Count + 2);
+        }
+
+        public override void Serialise(DataStreamWriter writer)
+        {
+            base.Serialise(writer);
+            writer.Write(RoundNumber);
+            writer.Write((byte) Players.Count);
+            foreach (var p in Players)
+            {
+                writer.Write(p);
+            }
+        }
+
+        public override void Deserialise(DataStreamReader reader, ref DataStreamReader.Context context)
+        {
+            RoundNumber = reader.ReadByte(ref context);
+
+            var playerCount = reader.ReadByte(ref context);
+            Players = new List<byte>();
```

Comment:
should the list maybe be initialised in constructor?

Reply:
Yeah, I think that's what we did elsewhere. Although I was considering in the future we might want to switch to arrays over lists (when efficiency becomes important)? Can switch for now though